src/core/Expander/ExpanderGroup.tsx

```diff
@@ -33,22 +33,21 @@ const StyledExpanderGroup = styled(
     <CompExpanderGroup {...passProps} />
   ),
 )`
-  ${props => baseStyles(props)};
+  ${(props) => baseStyles(props)};
 `;
 
 const OpenAllButton = ({
   children,
   ...passProps
 }: ExpanderOpenAllButtonProps) => {
-  const Component = React.Children.toArray(children)[0];
```

Comment:
This is another thing to test thoroughly. React.Children.toArray relies on JSX elements and the runtime type information of these is very limited. This broke after updating TypeScript and linters.

Reply:
This seems to work as intended. The typeof children is either object, string or null etc. So in case we provide a string or something not Suomifi Button, it is rendered inside unstyled button. Button is rendered as is.